src/controller/investments.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const userInvestments = require('../data/data');
+
+const investments = express.Router();
+investments.use(express.json());
+
+investments.get('/user/:uid', (req, res) => {
+  const { uid } = req.params;
+  const userData = userInvestments.find((data) => data.user_id === uid);
+  if (userData) {
+    return res.status(200).json(userData);
```

Comment:
You don't need to specify status code here and you can just use `res.send`
```suggestion
    return res.send(userData);
```